src/makeRobot.js

```diff
@@ -37,8 +37,73 @@
  *
  * @return {object}
  */
+
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    version: version,
+    wheels: wheels,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, chip version: \
+${this.version}, wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(value) {
+      if (value === undefined) {
+        this.coords.y++;
+      } else if (value > 0) {
+        this.coords.y = this.coords.y + value;
```

Comment:
Здесь было бы лучше написать 
```suggestion
        this.coords.y += value;
```
в остальных методах где нужно -=